core/src/data/tileData.h

```diff
@@ -63,7 +60,7 @@ enum GeometryType {
     polygons
 };
 
-typedef glm::vec3 Point;
```

Comment:
While we are on it and touching this file, can you change these to use `using`. Its just cosmetic, but thats what the standard is going with these days!

Reply:
Makes sense - done.